backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/report/baseandcustom/report.feature

```diff
@@ -0,0 +1,140 @@
+      | po_ic_1                | po1_l1         |
+    And after not more than 60s, C_Invoice_Candidates are not marked as 'to recompute'
+      | C_Invoice_Candidate_ID.Identifier |
+      | po_ic_1                           |
+    And process invoice candidates
+      | C_Invoice_Candidate_ID |
+      | po_ic_1                |
+    And after not more than 60s, C_Invoice are found:
+      | C_Invoice_ID      | C_Invoice_Candidate_ID |
+      | purchaseInvoice_1 | po_ic_1                |
+    And The jasper process is run
+      | Value                     | TableName | Identifier        |
+      | Eingangsrechnung (Jasper) | C_Invoice | purchaseInvoice_1 |
+
+
+
+  @S0471_200
+  @from:cucumber
+  Scenario: Sales Report Test
+    And metasfresh contains C_Orders:
+      | Identifier | IsSOTrx | C_BPartner_ID | DateOrdered |
+      | so1        | true    | customer      | 2021-04-16  |
+    And metasfresh contains C_OrderLines:
+      | Identifier | C_Order_ID | M_Product_ID | QtyEntered |
+      | so1_l1     | so1        | product      | 10         |
+    When the order identified by so1 is completed
+    And after not more than 60s, M_ShipmentSchedules are found:
+      | Identifier | C_OrderLine_ID | IsToRecompute |
+      | ss1        | so1_l1         | N             |
+    And The jasper process is run
+      | Value            | TableName | Identifier |
+      | Auftrag (Jasper) | C_Order   | so1        |
+    And 'generate shipments' process is invoked individually for each M_ShipmentSchedule
+      | M_ShipmentSchedule_ID | QuantityType | IsCompleteShipments | IsShipToday |
+      | ss1                   | D            | true                | false       |
+    And after not more than 60s, M_InOut is found:
+      | M_ShipmentSchedule_ID | M_InOut_ID |
+      | ss1                   | shipment1  |
+    And The jasper process is run
+      | Value                 | TableName | Identifier |
+      | Lieferschein (Jasper) | M_InOut   | shipment1  |
+    And after not more than 60s, C_Invoice_Candidate are found:
+      | C_Invoice_Candidate_ID.Identifier | C_OrderLine_ID.Identifier | QtyToInvoice |
+      | so_ic1                            | so1_l1                    | 10           |
+    When process invoice candidates and wait 60s for C_Invoice_Candidate to be processed
+      | C_Invoice_Candidate_ID |
+      | so_ic1                 |
+    Then after not more than 60s, C_Invoice are found:
+      | C_Invoice_Candidate_ID | C_Invoice_ID  |
+      | so_ic1                 | salesInvoice1 |
+    And validate created invoices
+      | C_Invoice_ID  | C_BPartner_ID | C_BPartner_Location_ID | processed | docStatus |
+      | salesInvoice1 | customer      | customerLocation       | true      | CO        |
+    And The jasper process is run
+      | Value             | TableName | Identifier    |
+      | Rechnung (Jasper) | C_Invoice | salesInvoice1 |
+
+
+
+
+  @from:cucumber
+  Scenario: Deactivate StoreArchiveOnFileSystem
```

Comment:
are the scenarios executed in order, so you know this will be last executed?

Reply:
Yes, scenarios in one file are executed in order, only order of feature files isn't fix. I did it in an sepereate scenario so it is also executed on failure